runners/spark/src/main/java/org/apache/beam/runners/spark/util/BroadcastHelper.java

```diff
@@ -31,97 +30,53 @@
 /**
  * Broadcast helper.
  */
-public abstract class BroadcastHelper<T> implements Serializable {
+public class BroadcastHelper<T> implements Serializable {
```

Comment:
The purpose of this class was not changed, only the constructor is from bytes and not T. I think there is no reason to change it. it is not tightly coupled to side input